Documentation/Installation.md

```diff
@@ -0,0 +1,43 @@
+
+# Installation
+
+<br>
+
+## Dependencies
+
+You will require `prettier`.
+
+<br>
+
+### PNPM
+
+If you use this method, simply add the <br>
+following code to your `.npmrc` config.
+
+```npmrc
+public-hoist-pattern[]=*
+```
+
+<br>
+
+### Manual
+
+Alternatively you can also install `prettier` <br>
+in your `devDependencies` by yourself.
+
+*If you are using `npm`, the prettier* <br>
+*dependency is hoisted automatically.*
+
+<br>
+<br>
+
+## Installing
+
+```shell
+npm install     \
+    --save-dev  \
+    prettier-config-atomic
```

Comment:
Make it one lineer. `\` implies bash

Reply:
I think it means to just write `npm install --save-dev prettier-config-atomic`, all on one line, no need to break it into seperate lines with `\`

Reply:
yeah, but because it is a pretty short command I think it is nicer to have it in one line. If there were multiple packages being installed, then I think I would agree that it makes more sense to break it up into separate lines

Reply:
I use Powershell and nushell which don't support `\`. 

> more readable

This is subjective. It is not more readable for me as an example.